### ISSUE TITLE
Report empty Paragraph markers

### ISSUE THREAD

**Issue by beniza** [Bridgeconn/usfm-grammar#45]
Currently, the grammar permits the use of repeated *empty* paragraph markers. This shouldn't be allowed as it doesn't make any sense. Please see the eg. below

```
\id HAB 45HABGNT92.usfm, Good News Translation, June 2003
\c 3
\s1 A Prayer of Habakkuk
\p
\p
\v 1 This is a prayer of the prophet Habakkuk:
\b
\q1
\m
\p
\p 
\q 
\s 
\p
\v 2 O \nd Lord\nd*, I have heard of what you have done,
\q2 and I am filled with awe.
\q1 Now do again in our times
\q2 the great deeds you used to do.
\q1 Be merciful, even when you are angry.
```

If this happens, the system should report it as an issue.

**Comment by joelthe1** [comment]
Thanks for the report @beniza. We are planning to add support for catching some of these as warnings in `usfm-grammar`.

**Comment by beniza** [comment]
Thank you

**Comment by joelthe1** [comment]
Fixed and merged via #94
